rootfs-builder/rootfs.sh

```diff
@@ -103,6 +107,14 @@ ENV PATH=\$PATH:\$GOROOT/bin:\$GOPATH/bin
 	popd
 }
 
+setup_agent_init() {
+	agent_bin="$1"
+	init_bin="$2"
+	info "Install $agent_bin as init process"
+	mv -f "${agent_bin}" ${init_bin}
```

Comment:
What about make a symlink from /sbin/init to the agent installed in bin?

Reply:
When kata agent is used as the init process, there is no need for `/sbin/agent` to be present.

Reply:
Yes, but doing this we avoid have an special case when agent is `init` for image/initrd building. Both both approaches sounds good.

Reply:
Actually the extra case helped me catch several places where `AGENT_INIT` env is not properly passed around. I hope to keep it to help future change as well.